src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/projects/ProjectShareSamplesIT.java

```diff
@@ -27,6 +27,8 @@ public void testShareSamplesAsManager() {
 		samplesPage.shareSamples();
 
 		assertFalse(shareSamplesPage.isNextButtonEnabled(), "");
+		shareSamplesPage.searchForProject("2");
```

Comment:
This does not really count as searching for a project by it's identifier since the project name also contains a 2.  What I would like to see is a completely different name from the identifier and confirm that the id is present in the tag.

Reply:
Please see 0d340a1f1dc2824f7a1b3a753e11c694c3875329.